wolox-books/src/views/Register.vue

```diff
@@ -31,7 +31,7 @@
   .sign-up-container
     button.main-button.text-xsmall(@click='submit' type='button')
       | Sign up
-  button.secondary-button.text-xsmall.white(type='button')
+   router-link.secondary-button.text-xsmall.white(:to='router.login')
```

Comment:
Change the destination to '/login'

Reply:
I talked to @BJordi and it should be written this way, as it is the one recommended in the vue docs:

`router-link(:to='{ name: routes.login}')`

Adapt 'name' in router.js to routes.login so it works with the new format